pkg/ccl/multitenantccl/tenantcostclient/tenant_side.go

```diff
@@ -121,14 +122,27 @@ const bufferTokens = 5000
 // magnitude as request units.
 const tokensPerCPUSecond = 1000
 
+// LocalityNetworkPath describes a source -> destination network flow.
+type LocalityNetworkPath struct {
```

Comment:
This is no longer used, now that we're incrementing the metric directly.

Reply:
Removed.